assignment8.txt

```diff
@@ -0,0 +1,13 @@
+Can Parlar
+
+1. printf means formatted input so you do not just print a text you can format the text by adding % signs to print numbers and such. fprintf has 2 formats one to print more than a text and one to print where. Normal printf prints to the standart output but fprintf can do it anywhere.
```

Comment:
sort of - your explanation is very wordy, but yeah, the fprintf prints to the specified stream, whereas printf prints to the default stream.